libraries/ms-common/inc/pwm.h

```diff
@@ -13,14 +13,17 @@
 #include "pwm_mcu.h"
 #include "status.h"
 
-// Initializes the PWM for a set timer with a period in milliseconds.
-StatusCode pwm_init(PWMTimer timer, uint16_t period_ms);
+// Initializes the PWM for a set timer with a period in microseconds.
+StatusCode pwm_init(PWMTimer timer, uint16_t period_us);
 
-// Gets the current period of a specified PWM timer.
+// Initializes the PWM for a set timer with a specific frequency (Hz)
+#define pwm_init_hz(timer, frequency) pwm_init(timer, 1000000 / frequency)
```

Comment:
Please move this above the function declaration (we keep #defines at the top of the file) and wrap macro parameters in braces. (See https://wiki.sei.cmu.edu/confluence/display/c/PRE01-C.+Use+parentheses+within+macros+around+parameter+names for explanation)